CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 0.12.0
+
+## Features
+
+- For the COPR projects created by packit via `copr-build` command are users able
+  to set more options (e.g. whether they will be visible on the homepage,
+  created as temporary)
```

Comment:
My suggestion:

Users are now able to set more options (e.g. whether they will be visible on the homepage, created as temporary) for COPR projects created via `copr-build` command.